test/archethic/mining/distributed_workflow_test.exs

```diff
@@ -914,6 +918,26 @@ defmodule Archethic.Mining.DistributedWorkflowTest do
             Workflow.add_cross_validation_stamp(coordinator_pid, stamp)
           end
 
+          receive do
+            {:ack_replication, sig, pub} ->
```

Comment:
Won't this case always be true and bottom receive will not work?

Reply:
Here you are awaiting the replication to be finished, once it's done you are notifying the validation process about the completion of the replication.
So we have to do it for all the storage nodes involved.